src/components/track-changes/suggestion-list/Suggestion.tsx

```diff
@@ -26,15 +26,15 @@ const scrollIntoView = (element: HTMLElement) => {
 }
 
 interface Props {
-  suggestion: TrackedChange
+  suggestions: TrackedChange[]
```

Comment:
as basically and TrackedChange now represented by an array, it may be a bit more semantically precise to include the array inside the type. i.e. to rename "TrackedChange" for example to be "InternalChange", and now use the name for TrackedChange = Array<InternalChange>, does it make sense?

Reply:
updated to the new type